CRM/Event/Form/ManageEvent/Fee.php

```diff
@@ -522,6 +522,11 @@ public static function formRule($values) {
           $errors['pay_later_receipt'] = ts('Please enter the Pay Later instructions to be displayed to your users.');
         }
       }
+      else {
+        if (empty($values['payment_processor'])) {
+          $errors['payment_processor'] = ts('You have indicated that this is a paid event, but no payment option has been selected. If this is not a paid event, please select the "No" option at the top of the page. If this is a paid event, please select at least one payment processor and/or enable the pay later option.');
+        }
+      }
```

Comment:
Tested and I can confirm this works well for Event Registration Page configuration.  I approve this change.